acceptance/acceptance_test.go

```diff
@@ -290,6 +290,10 @@ func getSkipReason(config *internal.TestConfig, configPath string) string {
 			return fmt.Sprintf("Disabled via RequiresUnityCatalog setting in %s (TEST_METASTORE_ID=%s)", configPath, os.Getenv("TEST_METASTORE_ID"))
 		}
 
+		if isTruePtr(config.RequiresWarehouse) && os.Getenv("TEST_DEFAULT_WAREHOUSE_ID") == "" {
+			return fmt.Sprintf("Disabled via RequiresWarehouse setting in %s (TEST_DEFAULT_WAREHOUSE_ID=%s)", configPath, os.Getenv("TEST_DEFAULT_WAREHOUSE_ID"))
```

Comment:
`TEST_DEFAULT_WAREHOUSE_ID` in the skip reason will always be empty, per the conditional.

Reply:
Same holds for the surrounding conditionals.

Reply:
addressed in #2620